cardano-config/src/Cardano/Config/Types.hs

```diff
@@ -248,20 +247,15 @@ parseNodeConfigurationFP :: ConfigYamlFilePath -> IO NodeConfiguration
 parseNodeConfigurationFP (ConfigYamlFilePath fp) = do
  nc  <- decodeFileThrow fp
  let genFile = unGenesisFile $ ncGenesisFile nc
- exists <- doesFileExist genFile
- case exists of
-   -- Genesis file is an absolute path
-   True -> pure nc
-   -- Genesis file is a relative path (relative to configuration yaml filepath)
-   False -> do let d = takeDirectory fp
-               pure $ nc { ncGenesisFile = GenesisFile $ d </> genFile }
-
--- TODO: Make genesisfile relative to configuration file as above.
+ -- Make genesis file relative to configuration yaml filepath.
+ let d = takeDirectory fp
+ pure $ nc { ncGenesisFile = GenesisFile $ d </> genFile }
+
 parseNodeConfiguration :: NodeProtocolMode -> IO NodeConfiguration
 parseNodeConfiguration npm =
   case npm of
-    MockProtocolMode (NodeMockCLI _ _ cy _) -> decodeFileThrow $ unConfigPath cy
-    RealProtocolMode (NodeCLI _ _ cy _) -> decodeFileThrow $ unConfigPath cy
+    MockProtocolMode (NodeMockCLI _ _ cy _) -> parseNodeConfigurationFP cy
```

Comment:
This is also why it was not working before :man_facepalming: I needed to update `parseNodeConfiguration` with `parseNodeConfigurationFp`